### DISCUSSION TITLE
Group multiple tabs of Preview.app in a single window

### DISCUSSION THREAD

**Discussion by abrihaye** [nikitabobko/AeroSpace#758] [questions-and-answers]
Hello everyone,
I am currently getting started with aerospace. My preview app on MacOS uses multiple tabs. 
It looks like each tab is seen as a single window in my configuration. That means each tab is its "own" tile, and once I switch to a different tab, my window moves to another position on the screen.

I would like to group all my tabs from a single Preview.app as a single tile from the aerospace point of view. 
Would you have any recommendations?

Thank you in advance

**Comment by jeofo** [comment]
Roughly the same issue here, Aerospace allocates space for each of the tabs though it's all collated in one Preview.app window. So when I open `n` files with Preview, there's `n-1` blank spaces on my screen and only one is allocated to the currently opened Preview app.

**Comment by Austin4705** [comment]
I'm also getting this issue. Any fixes would be appreciated

**Comment by zhenpewu** [comment]
https://github.com/nikitabobko/AeroSpace/issues/68

**Reply by sl4mmy** [reply]
The discussion for #68 indeed has a work-around, however I believe that Preview.app automatically opens new documents in tabs when a certain number of documents are already opened.  For example, I believe that by default you can have up to 10 documents open, each in their own window, but when you open the 11th document it automatically opens as a new tab in one of the existing windows.

I believe I understand the limitations discussed in #68 and why this issue is difficult to solve with current macOS APIs (although I am encouraged by Nikita's last comment in that discussion), I'm just pointing this out as a known limitation of the suggested work-around.
